.github/workflows/build.yml

```diff
@@ -32,4 +32,4 @@ jobs:
         ANDROID_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD }}
       with:
         lane: build
-        options: '{ "flavor": "unbranded" }'
+        options: '{ "flavor": "icm_ph_chc" }'
```

Comment:
There is no need to replace the default flavor here.

Reply:
Thanks, I have reverted the change.